### DISCUSSION TITLE
Pré lançamento

### DISCUSSION THREAD

**Discussion by murara** [updatedev-com/gatsby-markdown-blog-starter#11] [Ideas]
Acredito que precisamos resolver algumas coisas antes de lançar o site, seguem as ideias:

**Redes Sociais**

- Ocultar botões de redes sociais do site, pois ainda não temos. (Issue #12)

**Páginas para Criar**

- About (https://github.com/updatedev-com/updatedev.com/issues/6)
- ~~How to post (contribute)~~ _para depois_
- ~~Donate 😆 ~~ _para depois_
- Contact (https://github.com/updatedev-com/updatedev.com/issues/7)

**Google Analytics & SEO** (https://github.com/updatedev-com/updatedev.com/issues/8)

- Setup
- Revisão SEO

**Comment by kelvinoenning** [comment]
Acho que esse `How to post (contribute)` seria interessante a gente criar só quando o blog estiver como repo público, pra galera poder saber como publicar, dado o resto realmente precisamos fazer.

**Comment by murara** [comment]
Concordo com você. Podemos deixar pra depois.

**Comment by murara** [comment]
@kelvinoenning tem um lance que esqueci: setup do google analytics

**Comment by murara** [comment]
@kelvinoenning deixei para depois o lance de donate, ja criei as issues no outro projeto usando a tag `pre-launch` para sabermos tudo que tem para fazer antes do lançamento.
